website/docs/cluster-management/getting-started.mdx

```diff
@@ -211,6 +211,19 @@ Then update the `GITOPS_REPO` variable to point to your cluster
   {CapiGitopsCDC}
 </CodeBlock>
 
+#### Add Monitoring Dashboards to your cluster
+
+In order to add dashboards to your cluster, you'll need to use metadata annotations following the below pattern.
+
+```
+apiVersion: gitops.weave.works/v1alpha1
+kind: GitopsCluster
+metadata:
+  annotations:
+    metadata.weave.works/dashboard.grafana: https://grafana.com/
```

Comment:
Are you talking about specifically in the part post forward slash? The issue we have up about reformatting does not include a case with `.` characters, so I'm assuming it would just display as is. Is that bad?

Reply:
With @ozamosi's explanation, I understood now that this metadata was for different resource kinds, not those we were working on in Core.